Common/Source/Dialogs/dlgWayQuick.cpp

```diff
@@ -132,7 +133,7 @@ short dlgWayQuickShowModal(void){
   if (WPLSEL.Format == LKW_CUP) {
         TCHAR ttmp[50];
         // and it is landable
-        if ((WPLSEL.Style>1) && (WPLSEL.Style<6) ) {
+        if ((WPLSEL.Style >= STYLE_AIRFIELDGRASS) && (WPLSEL.Style <= STYLE_AIRFIELDSOLID) ) {

```

Comment:
no problem, it's more clear like that STYLE_AIRFIELDGRASS are first landable and STYLE_AIRFIELDSOLID are last landable.